agent/test/README.md

```diff
@@ -0,0 +1,46 @@
+# Test for HeapStats
+
+Test cases for HeapStats. These test cases could run on containers.
+
+## Usage
+
+### on container
+
+```
+docker build --pull -t heapstats/$(basename $PWD) -f ./Dockerfile(.el7) .
```

Comment:
Please add description for Red Hat subscription.
I guess we should have valid Red Hat subscription(s) to use the docker registry in Red Hat.